impeller/aiks/experimental_canvas.cc

```diff
@@ -391,6 +391,7 @@ void ExperimentalCanvas::DrawTextFrame(
   text_contents->SetForceTextColor(paint.mask_blur_descriptor.has_value());
   text_contents->SetScale(GetCurrentTransform().GetMaxBasisLengthXY());
   text_contents->SetColor(paint.color);
+  text_contents->SetOffset(position);
```

Comment:
That looks necessary. I'm confused as to why this version has a call to `SetScale` while the regular canvas version does not. Is that another issue?

Reply:
For the regular canvas, we populate the scale when populating the glyph atlas:

https://github.com/flutter/engine/blob/main/impeller/entity/contents/text_contents.cc#L66-L71

Reply:
Ah, and for the experimental canvas that is done before the TextContents exists. I see...